### ISSUE TITLE
New Feature request

### ISSUE THREAD

**Issue by danilopatro** [Sydtrack/ringsdb#20]
Add to smart filter a new criteria "m" or any other letter, to retrieve just the cards from "My collection".

**Comment by staurasi** [comment]
Yes, please. It totally make sense and would be a step forward compared to other Lotr LCG card search engines (i.e. hallofbeorn) that doesn't allow you to save your collection.

**Comment by staurasi** [comment]
@danilopatro You can build a long search string based on you collection piping you adventures packs/cycles/deluxes (e.g. this is mine: c:Core|SoM|KD|DD|HoN|TH|TSF)

**Comment by danilopatro** [comment]
Great idea @staurasi! (this is mine: **_e:_Core|HfG|CatC|JtR|HoEM|TDM|RtM|OHaUH|KD**)

**Comment by staurasi** [comment]
Thanks @danilopatro. You just made me notice that we can't mix single expansions (**e:**) and cycles (**c:**) in one string. Thus you have to insert all single expansion one by one in a huge string if you doesn't own full cycles. Then my correct string sadly is:
**e:Core|HfG|CatC|JtR|HoEM|TDM|RtM|KD|TRG|RtR|WitW|TLD|FoS|SaF|OHaUH|OtD|HoN|TSF**

I hope that a sort of  "Restrict search to my collection" flag will come soon.

**Comment by staurasi** [comment]
Another temporary workaround is to create an empty "search deck" and search for cards using filters in deck edit mode.
